osf/models/notifications.py

```diff
@@ -103,4 +103,4 @@ class NotificationDigest(ObjectIDMixin, BaseModel):
     event = models.CharField(max_length=50)
     message = models.TextField()
     # TODO: Could this be a m2m with or without an order field?
-    node_lineage = ArrayField(models.CharField(max_length=5))
```

Comment:
This little bit of code here is responsible for the whole migration, because QFNs have 10 digit guids.